webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -418,7 +420,12 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
         doc.newPage();
     }
 
-    private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, ScrambleRequest scrambleRequest, String globalTitle, int index) throws DocumentException, IOException {
+    private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, ScrambleRequest scrambleRequest, String globalTitle, int index, Locale locale) throws DocumentException, IOException {
+        
+        // internationalization
+        Locale currentLocale = new Locale(locale.getLanguage(), locale.getCountry());
+        ResourceBundle messages = ResourceBundle.getBundle("net.gnehzr.tnoodle.server.webscrambles.Internationalization.MessagesBundle", currentLocale);
```

Comment:
it looks this code still suffers from the unicode encoding issue. I proposed a fix for that here: https://github.com/thewca/tnoodle/pull/242#issuecomment-344471075.

Reply:
Fixed